pkg/koordlet/audit/event_logger.go

```diff
@@ -212,6 +227,10 @@ func listAuditfiles(dir string) ([]string, error) {
 	pattern := fmt.Sprintf("%s/audit-*.log", dir)
 	files, err := filepath.Glob(pattern)
 	if err == nil {
+		// reverse files
```

Comment:
hi, what is the purpose to reverse these files

Reply:
While reading audit files, we expect the audit files order by `date` desc, and the `Glob function` will do the asc by default.   @zwzhang0107

Reply:
Another question, should we add a filter flag in `CLI` to discard any events?

Reply:
now i see the result is already desc, or do you want to get the event from old to new(asc)? maybe add a query param to control this like (desc=true by deafult/false)
![image](https://user-images.githubusercontent.com/101699117/210481161-fa3a37af-d0a9-4f31-b937-877e1b0e621d.png)

Reply:
In one file the result is correct, and returned content by `desc` order. But, if we have N files? 

For example, we have those files in the audit log directory: `audit.log`、`audit-2023-01-04-14-28-01.log`、`audit-2023-01-04-13-28-01.log`、`audit-2023-01-04-12-28-01.log`.

 the `Glob function` will return by this order: [`audit-2023-01-04-12-28-01.log`, `audit-2023-01-04-13-28-01.log`, `audit-2023-01-04-14-28-01.log`], when read logger finished `audit.log`, we expect the next file is `audit-2023-01-04-14-28-01.log` which is the latest audit files expect `audit.log`.   @zwzhang0107

Reply:
i get your idea, this looks great, good job!!